website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+---
+title: '0.72 - Beta Support for Package Exports and Symlinks'
+authors: [kelset, fortmarek, lunaleaps, cipolleschi]
+tags: [announcement, release]
+date: 2023-05-30
+---
+
+Today we’re releasing 0.72!
+
+This release adds a couple of highly requested features for Metro and developer experience improvements. So much of this work stems from your feedback on the [2022 community survey](https://github.com/react-native-community/discussions-and-proposals/discussions/528) -- thank you to all those that participated!
+
+### Highlights
+
+- [Metro support for Package Exports, symlinks, and config changes](/blog/2023/05/30/0.72-metro-package-exports-symlinks#metro)
+- [Developer experience improvements](/blog/2023/05/30/0.72-metro-package-exports-symlinks#developer-experience-improvements)
+- [Moving New Architecture updates](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+- [Mitigation for TextInput ANRs on Android](/blog/2023/05/30/0.72-metro-package-exports-symlinks#react-native-package-renames)
+
+### Breaking Changes
+
+- [Deprecated Component Removals](/blog/2023/05/30/0.72-metro-package-exports-symlinks#deprecated-component-removals)
+- [Package renames](/blog/2023/05/30/0.72-metro-package-exports-symlinks#package-renames)
+
+<!--truncate-->
+
+## Highlights
+
+### Metro
```

Comment:
I intentionally left it this way because as a bullet, it makes sense to describe what we're covering in a succinct way (because a reader may not have clicked in) but as a title we have the following sub-titles to clarify --

Reply:
let's find a title that is a "middle ground" then, so that it can be consistent?